### ISSUE TITLE
No output in tree.py

### ISSUE THREAD

**Issue by mansiag** [OpenGenus/cosmos#3441]
**This is a(n):**

- [ ] New algorithm
- [ ] Update to an existing algorithm
- [x] Error
- [ ] Proposal to the Repository

**Details:**
[tree.py](https://github.com/OpenGenus/cosmos/blob/master/code/data_structures/src/tree/binary_tree/binary_tree/tree/tree.py) does not give any output.

**Comment by hrsvrdhn** [comment]
Is anyone working on this issue?

**Comment by mansiag** [comment]
You can work on this issue.

**Comment by kobra-cai** [comment]
I don't think this has necessarily been solved so I'll make a pull-request that has modified the file so that it can print specific insertions, deletions, and print the tree itself.

Explanation:
For example, it takes in an input_file.txt containing i6 i8 i3 i1 i9 i5 i2 d1 d5 p. 
i6 means insert 6, d1 means delete 1, and p means print the nodes. The printing in this case uses the traversal in the original implementation and outputs the nodes and their relations under the "BST representation" heading. Example below.

The output of the above test case would be: 
root: 6
Insert: 8
8 goes right (parent: 6)
Insert: 3
3 goes left (parent: 6)
Insert: 1
1 goes left (parent: 6)
1 goes left (parent: 3)
Insert: 9
9 goes right (parent: 6)
9 goes right (parent: 8)
Insert: 5
5 goes left (parent: 6)
5 goes right (parent: 3)
Insert: 2
2 goes left (parent: 6)
2 goes left (parent: 3)
2 goes right (parent: 1)
Delete: 1
Delete: 5


BST representation
6
left next level: 3
right, next level: 8
3
left next level: 2
8
right, next level: 9
2
leaf
9
leaf
